ref/general/health-check-microservices.adoc

```diff
@@ -25,15 +25,15 @@ A self-check can assess anything that the service needs:
 
 Services report as either `UP` (available) or `DOWN` (unavailable) by implementing the API provided by MicroProfile Health. A service orchestrator can then use these status reports to decide how to manage and scale the services within an application. Health checks can also interact directly with link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/[Kubernetes liveness and readiness probes].
 
-For example, in a microservices-based banking application, health checks might be implemented on the login service, the balance transfer service, and the bill pay service. If a health check on the balance transfer service detects a bug or deadlock, it reports as DOWN. In this case, if the `/health/live` endpoint is configured in a Kubernetes liveness probe, the probe restarts the service automatically, saving the user from encountering downtime or an error and preserving the functions of the other services in the application.
+For example, in a microservices-based banking application, you might implement health checks on the login service, the balance transfer service, and the bill pay service. If a health check on the balance transfer service detects a bug or deadlock, it reports as DOWN. In this case, if the `/health/live` endpoint is configured in a Kubernetes liveness probe, the probe restarts the service automatically. This feature saves the user from encountering downtime or an error and preserves the functions of the other services in the application.
```

Comment:
I agree that breaking the last sentence into two shorter ones makes sense but I don't think "This feature saves..." is any more informative than just saying "This saves..." (which is I'm guess the reason for saying "feature". The word "feature" is overloaded and is just used here as a filler. Instead, could reword the second sentence to start "Restarting the service automatically saves the user..."?